notification-server/src/bin/service.rs

```diff
@@ -93,9 +93,9 @@ struct Args {
     )]
     block_process_timeout_sec: u64,
     #[arg(
-        long = "listen-address",
-        help = "Listen address for the server.",
-        env = "NOTIFICATION_SERVER_METRICS_LISTEN_ADDRESS"
+        long = "prometheus-address",
+        help = "Listen address for the prometheus metrics server.",
+        env = "NOTIFICATION_SERVER_PROMETHEUS_ADDRESS"
```

Comment:
Is it not an issue that these are the same if they are run on the same machine?

Reply:
Depends on how they are ran indeed. 

If they are ran as part of the docker compose then it obviously not be a problem.

If both services are ran on localhost using the same environment variable then indeed it would. 

However I did not update the env.template, which means that if you run using the configuration as is everything will work just fine. This also means that if one is to init the applications, the suggested way would be to use arguments instead of variables.

I agree with what you are writing, however I would prefer being consistent here, and by consistent I mean that for the rest environment variables used as part of config of both applications they are named the same.

As part of writing this comment I found a few issues in the makefile and README. :)